include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -3436,9 +3436,7 @@
       "opcode" : 5632,
       "operands" : [
         { "kind" : "IdRef",         "name" : "'Target'" },
-        { "kind" : "Decoration" },
-        { "kind" : "LiteralString" },
-        { "kind" : "LiteralString", "quantifier" : "*" }
+        { "kind" : "Decoration" }
```

Comment:
Right.  In the grammar file the Decoration enum lists its own parameters.

Reply:
Yes. The consumer of the grammar should further consult the grammar of the concrete decoration to determine how to proceed. This is because each decoration can have its own list of parameters, varied of types, we cannot easily represent that here.

Reply:
Sorry. It was my mistake that I forgot about this. `OpDecorate` and friends are all defined likewise.

Reply:
I see now, it follows OpDecorate. Thanks. The first PR certainly looked good. :)